passt/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest
+
+ENV CNI_PATH="/cni"
+RUN apk add --no-cache bash
+
+ARG KUBEVIRT_VERSION
+
+RUN mkdir -p ${CNI_PATH}
+
+ARG BINARY_URL=https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-passt-binding
```

Comment:
We don't need the BINARY_URL as argument

Reply:
we do need it as ARG, because this way ADD will able to use it, unless it is embedded on the ADD command 
anyhow you asked that it will be external ARG

Reply:
about adding `BINARY_URL` as input ARG that we spoke offline
we still dont support nightly, and we still dont even know what is the nighly URL exactly of the binary
i think it is not correct to add it right now as it is half baked, once we finish this cycle and have the CNI all the flow, we can add whatever we need, wdyt?

Reply:
i will inline it